packages/web3.js-plugin-fee-estimation/src/plugin.spec.ts

```diff
@@ -0,0 +1,241 @@
+import { beforeAll, describe, expect, test } from 'vitest'
+import Web3, { Contract, FMT_BYTES, FMT_NUMBER } from 'web3'
+import {
+  l2StandardBridgeABI,
+  l2StandardBridgeAddress,
+  optimistABI,
+  optimistAddress,
+} from '@eth-optimism/contracts-ts'
+
+import OptimismFeeEstimationPlugin from './plugin'
+
+const provider = process.env.VITE_L2_RPC_URL ?? 'https://mainnet.optimism.io'
```

Comment:
```suggestion
import {z} from 'zod'
const defaultProvider = 'https://mainnet.optimism.io'
const provider = z.string().url(process.env.VITE_L2_RPC_URL).default(defaultProvider)
if (provider === defaultProvider) console.warn('warning: using default public provider. Tests may face rate limits. To pass in a different provider set VITE_L2_RPC_URL in env')
```
Nit would prefer strict validation like this